src/base/datetime/date/index.ts

```diff
@@ -85,10 +82,7 @@ export class BaseDate extends KucBase {
         aria-haspopup="dialog"
         aria-expanded="${this._dateTimeCalendarVisible}"
         class="kuc-base-date__assistive-text"
-        @keydown="${this._handleKeyDownButton}"
-        @focus="${this._handleFocusButton}"
-        @blur="${this._handleBlurButton}"
-        ?disabled="${this.disabled}"
+        @click="${this._handleClickButton}"
```

Comment:
when remove @keydown event, you removed ?disabled too => So please add ?disabled.